receiver/kafkametricsreceiver/broker_scraper.go

```diff
@@ -63,6 +218,25 @@ func (s *brokerScraper) scrape(context.Context) (pmetric.Metrics, error) {
 
 	brokers := s.client.Brokers()
 
+	allMetrics := make(map[string]map[string]interface{})
+
+	if s.saramaConfig != nil {
+		allMetrics = s.saramaConfig.MetricRegistry.GetAll()
+	}
+
+	for _, broker := range brokers {
+		broker := broker
+		s.scrapeConsumerFetch(allMetrics, broker)
+		s.scrapeIncomingByteRate(allMetrics, broker)
+		s.scrapeOutgoingByteRate(allMetrics, broker)
+		s.scrapeRequestLatency(allMetrics, broker)
+		s.scrapeRequestRate(allMetrics, broker)
+		s.scrapeRequestSize(allMetrics, broker)
+		s.scrapeRequestsInFlight(allMetrics, broker)
+		s.scrapeResponseRate(allMetrics, broker)
+		s.scrapeResponseSize(allMetrics, broker)
+	}
+
 	s.mb.RecordKafkaBrokersDataPoint(pcommon.NewTimestampFromTime(time.Now()), int64(len(brokers)))
```

Comment:
let's define `now` once and pass it to all the functions so all the metrics have the same timestamp